lib/asciidoctor-pdf/theme_loader.rb

```diff
@@ -86,8 +86,10 @@ def load hash, theme_data = nil
   private
 
   def process_entry key, val, data
-    if key == "admonition_icons"
-      data[key] = symbolize_keys val
+    if key.start_with? 'admonition_icon_'
+      data[key] = ::Hash[(val || {}).map {|(key2, val2)|
```

Comment:
I agree. I rewrote it slightly for clarity.

I really wish that calling map on a Hash returned a Hash. One of the few things that really annoys me about Ruby. I had written it using curly braces to make that look better. I changed it to invoke `to_h` on the `end`. Still not pretty, but it will have to do. I try to avoid local variables where possible because they slow down Ruby (though in this case, makes no difference at all).